modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptRxjsClientCodegen.java

```diff
@@ -250,38 +253,65 @@ private void updateOperationParameterEnumInformation(Map<String, Object> operati
         operations.put("hasEnums", hasEnums);
     }
 
+    private void setParamNameAlternative(CodegenParameter param, String paramName, String paramNameAlternative) {
+        if (paramName == param.paramName) {
```

Comment:
About `==`, did you mean to use `.equals()` instead to compare the actual string value?